src/dune.in

```diff
@@ -2,7 +2,16 @@
  (name mathcomp.multinomials)
  ;coq-native-disabled; (mode native)
```

Comment:
```suggestion
```

Reply:
This comment can be removed now.

Reply:
@Alizter At the very least, removing this comment line should happen at the same time as removing the configure script.